src/window/window_wrapper/mod.rs

```diff
@@ -322,7 +322,14 @@ impl GlutinWindowWrapper {
                     },
                 ..
             } => self.handle_mouse_wheel(x as i32, y as i32),
-
+            Event::WindowEvent {
+                event:
+                    WindowEvent::MouseWheel {
+                        delta: MouseScrollDelta::PixelDelta(lpos),
+                        ..
+                    },
+                ..
+            } => self.handle_mouse_wheel(0, (lpos.y as i32)/10),
```

Comment:
The value given is (x, y). Horizontal scroll is disabled (hence the 0), Y is divided by 10 as the default was quite fast (jumping around 200-300 lines at a time)